packages/playwright-core/src/utils/comparators.ts

```diff
@@ -17,26 +17,17 @@
 
 import { colors, jpegjs } from '../utilsBundle';
 import pixelmatch from '../third_party/pixelmatch';
+import { compare } from '../image_tools/compare';
```

Comment:
Perhaps we should move this file (`comparators.ts`) to `image_tools` as well?

Reply:
I'd keep `comparators.ts` where they are since they combine all kinds of comparators (i.e. text comparators + different image comparators).